language-server/language-server-api/.js/src/main/scala/aqua/lsp/ResultHelper.scala

```diff
@@ -130,9 +130,9 @@ object ResultHelper extends Logging {
     CompilationResult(
       errors.toJSArray,
       warnings.toJSArray,
-      locationsToJs(lsp.variables.flatMap(v => v.allLocations)),
+      locationsToJs(lsp.variables.allLocations),
       importTokens,
-      tokensToJs(lsp.variables.map(_.definition))
+      tokensToJs(lsp.variables.variables.values.flatMap(_.map(_.definition)).toList)
```

Comment:
Maybe it is better to make it a method in `Variables`? Something like `lazy val definitions: List[...] = ...`.